scipy/optimize/minpack.py

```diff
@@ -500,8 +513,8 @@ def curve_fit(f, xdata, ydata, p0=None, sigma=None, absolute_sigma=False,
         The model function, f(x, ...).  It must take the independent
         variable as the first argument and the parameters to fit as
         separate remaining arguments.
-    xdata : An M-length sequence or an (k,M)-shaped array
-        for functions with k predictors.
+    xdata : An M-length sequence or an (k,M)-shaped array for functions with k
+        predictors.
```

Comment:
It didn't quite fix it. 

I think it should be just:
"An M-length sequence or an (k,M)-shaped array for functions with k predictors" (in one line, no period).

It must be really the last thing in this PR.